src/components/more-blogs-section/more-blogs-section.jsx

```diff
@@ -57,3 +59,8 @@ MoreBlogsSection.propTypes = {
     })
   })
 };
+
+MoreBlogsSection.defaultProps = {
+  refMoreFrom: {current: ''},
```

Comment:
Done

Reply:
Because, current is null in the more-blogs sections, but in the mdx file it is a div. I tried to use instanceOf(HTMLDivElement) but it gives error in the logs. What is the correct value for current?

Reply:
You should look for the correct value. The first thing you think: is that the `refMoreFrom` always has a key with value current? If always has this key, the second thing you think: what does current have in this key?